modules/dxp/apps/commerce-avalara/commerce-avalara-connector/src/main/java/com/liferay/commerce/avalara/connector/internal/CommerceAvalaraConnectorImpl.java

```diff
@@ -47,6 +49,18 @@
 )
 public class CommerceAvalaraConnectorImpl implements CommerceAvalaraConnector {
 
+	@Override
+	public List<TaxCodeModel> listTaxCodes() throws Exception {
+		AvaTaxClient avaTaxClient = _getAvaTaxClient();
+
+		// Check if need to paginate, docs say 1000 results max per call
```

Comment:
Need to remove comment. After running the job, 4,937 taxCategories were created.

Reply:
Is this still an open point?

Reply:
Removed comment - after running the job, 4,937 taxCategories were created so this is not an issue it seems.